app/uk/gov/hmrc/agentservicesaccount/views/pages/updatecontactdetails/email_locked.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
```

Comment:
Note: this view was lifted from agent-[overseas]-subscription
It was not part of the ticket spec but designers always forget that we need this view when we implement email verification